dom.bs

```diff
@@ -2490,6 +2492,22 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
    </li>
   </ol>
 
+ <li><p>If <var>textChanged</var> is true, run the <a>child text content change steps</a> for
```

Comment:
then run*

(so it's observable if we run these before the insertion steps? I guess there's a test?)

Reply:
It's not observable, just thought it would be better to specify them as running only once, otherwise for example if you follow the spec pedantically you would recreate a `style`'s stylesheet once for each added `Text` child.